develop/interoperability/send-messages.md

```diff
@@ -0,0 +1,44 @@
+---
+title: XCM - Send Messages
+description: Unlock blockchain interoperability with XCM — Polkadot's Cross-Consensus Messaging format for cross-chain interactions.
+---
+
+# XCM - Send Messages
+
+## XCM Frame Pallet Overview
+
+The XCM pallet [(pallet-xcm)](https://github.com/paritytech/polkadot-sdk/blob/master/polkadot/xcm/pallet-xcm/src/lib.rs) provides a set of pre-defined, commonly used XCVM programs in the form of a set of extrinsics using [FRAME](https://docs.substrate.io/reference/frame-pallets/).
+
+This pallet provides some default implementations for traits required by XcmConfig. The XCM executor is also included as an associated type within the pallet's configuration. For further details on XCM Configurations, see the previous page here.
+
+Where the XCM format defines a set of instructions used to construct XCVM programs, pallet-xcm defines a set of extrinsics that can be utilized to build XCVM programs, either to target the local or external chains. pallet-xcm's functionality is separated into three categories:
+
+**Primitive** - Dispatchable functions to locally execute an XCM
+
+**High-level** - Dispatchable functions for asset transfers
+
+**Version negotiation-specific** - Dispatchable functions
+
+This page will highlight the two Primary Primitive Extrinsics which are responsible for sending and executing XCVM programmes as dispatchable functions within the pallet.
+
```

Comment:
maybe we can complement this intro with something at a high level about the roles of pallet-xcm, just like: https://polkadot-blockchain-academy.github.io/pba-book/xcm/pallet/page.html#key-roles-of-pallet-xcm